src/Microsoft.OpenApi/Properties/SRResource.resx

```diff
@@ -210,6 +210,12 @@
   <data name="Validation_RuleAddTwice" xml:space="preserve">
     <value>The same rule cannot be in the same rule set twice.</value>
   </data>
+  <data name="Validation_SchemaDoesntContainDiscriminatorProperty" xml:space="preserve">
```

Comment:
>Validation_SchemaDoesntContainDiscriminatorProperty [](start = 14, length = 51)

Suggest renaming this to:

Validation_CompositeSchemaMustContainPropertySpecifiedInTheDiscriminator